test/conf/cdc.yaml

```diff
@@ -1,4 +1 @@
 cdc_enabled: true
-# Compression enabled since uncompressed + cdc isn't compatible w/Windows
-commitlog_compression:
```

Comment:
What do we think is the most common config? I'd think compression on and probably lz4 because it's legit. If so, we should test cdc w/the expected common parameters. i.e. I'd leave this here on that grounds.

Reply:
"default" in cassandra.yaml is uncompressed. I doubt people are using compressed commit logs more often as it is not default.